container/containerd/factory.go

```diff
@@ -31,7 +31,8 @@ import (
 	"github.com/google/cadvisor/manager/watcher"
 )
 
-var ArgContainerdEndpoint = flag.String("containerd", "unix:///var/run/containerd.sock", "containerd endpoint")
+var ArgContainerdEndpoint = flag.String("containerd", "/run/containerd/containerd.sock", "containerd endpoint")
```

Comment:
wait, does this still work if it isn't a unix socket?

Reply:
It has to be a unix socket because the original implementation of 'func Client' has following code ([see](https://github.com/0902horn/cadvisor/blob/150c78bd124181c983f86b423c6877fab1451ff7/container/containerd/client.go#L58)):
```
tryConn, err := net.DialTimeout("unix", address, connectionTimeout)
```
I think it is by design too. See this containerd [issue](https://github.com/containerd/containerd/issues/1324)

Reply:
I meant to ask if this will still connect to the unix socket if you remove the `unix://` prefix

Reply:
Oh...I thought you were asking TCP or something.
ArgContainerdEndpoint was not used before, so don't worry about changing its value's format. Now it is used and requires such format without `unix://` prefix.